### PR TITLE
enable further codeql scanning

### PR DESCRIPTION
**Pull request by klwhaley**
Signed-off-by: Kathryn Whaley <kathrynl@merative.com>

### CODE REVIEW

File: src/test/java/io/github/linuxforhealth/hl7/segments/Hl7EncounterFHIRConversionTest.java [hunk 1]
```diff
@@ -353,7 +353,7 @@ void testEncounterReasonCode() {
 
         CodeableConcept encounterReasonEVN = reasonCodes.get(0);
         CodeableConcept encounterReasonPV2 = reasonCodes.get(1);
-        if (encounterReasonPV2.getTextElement().toString() != "Fatigue") {
+        if (!encounterReasonPV2.getTextElement().toString().equalsIgnoreCase("Fatigue")) {
```
Comment (LisaWellman) [inline src/test/java/io/github/linuxforhealth/hl7/segments/Hl7EncounterFHIRConversionTest.java:356]: Do we want to ignore case?  Original check did not

Comment (LisaWellman) [reply src/test/java/io/github/linuxforhealth/hl7/segments/Hl7EncounterFHIRConversionTest.java:356]: Might as well fix the one on line 379 at the same time - same solution

File: src/test/java/io/github/linuxforhealth/hl7/segments/Hl7NoteFHIRConverterTest.java [hunk 2]
```diff
@@ -223,7 +223,7 @@ void testMedicationRequestNoteCreation(String message, String medicalRequestSegm
         Observation obsGlucose = ResourceUtils.getResourceObservation(observations.get(0), ResourceUtils.context);
         Observation obsHemoglobin = ResourceUtils.getResourceObservation(observations.get(1), ResourceUtils.context);
         // Figure out which is first and reassign if needed for testing
-        if (obsGlucose.getCode().getText() != "MEAN BLOOD GLUCOSE") {
+        if (!obsGlucose.getCode().getText().equalsIgnoreCase("MEAN BLOOD GLUCOSE")) {
```
Comment (LisaWellman) [inline src/test/java/io/github/linuxforhealth/hl7/segments/Hl7NoteFHIRConverterTest.java:226]: Do we want the check to be case-insensitive?  It was not before

Comment (LisaWellman) [reply src/test/java/io/github/linuxforhealth/hl7/segments/Hl7NoteFHIRConverterTest.java:226]: Might as well fix the one on line 108 at the same time - same solution

File: src/test/java/io/github/linuxforhealth/hl7/segments/Hl7EncounterFHIRConversionTest.java (hunk 1 above)
Comment (klwhaley) [reply src/test/java/io/github/linuxforhealth/hl7/segments/Hl7EncounterFHIRConversionTest.java:356]: fixed

**Review by LisaWellman — APPROVED** [review]

**Pull request merged**